app/app.go

```diff
@@ -620,6 +606,22 @@ func NewApp(
 		&savingsKeeper,
 		communitytypes.ModuleAccountName,
 	)
+
```

Comment:
nit: code comment for future readers on the required state / dependency order in case of future updates to that state / related state functions